cmd/call-endpoint/main.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/alecthomas/kong"
+
+	"github.com/edulinq/autograder/internal/api"
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/cmd"
+	"github.com/edulinq/autograder/internal/config"
+	"github.com/edulinq/autograder/internal/log"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+var args struct {
+	config.ConfigArgs
+	cmd.CommonOptions
+
+	Endpoint   string   `help:"Endpoint of the desired API." arg:""`
+	Parameters []string `help:"Parameter for the endpoint in the format 'key:value', e.g., 'id:123'." arg:"" optional:""`
+	Table      bool     `help:"Attempt to output data as a TSV. Will default to JSON." default:"false"`
```

Comment:
The second sentence ("Will default to JSON.") needs some work.
The parameter itself defaults to `false`, so your wording could be a bit confusing.
Since you use the word "attempt" in the previous sentence, you want this one to tell what happens when that attempt can't happen.